cc_licenses/templates/includes/legalcode_10_license.html

```diff
@@ -0,0 +1,68 @@
+           </li>
+
+        {# 4 #}
+<li class="padding-{% start %}-normal padding-bottom-normal">
+          <p><strong>{% blocktrans trimmed %}Limitations and Disclaimers{% endblocktrans %}</strong>.</p>
+
+<ol type="a" class="padding-{% start %}-normal padding-vertical-normal">
+<li class="padding-{% start %}-normal padding-bottom-small">{% blocktrans trimmed %}No trademark or patent rights held by Affirmer are waived, abandoned, surrendered, licensed or otherwise affected by this document.{% endblocktrans %}</li>
+<li class="padding-{% start %}-normal padding-bottom-small">{% blocktrans trimmed %}Affirmer offers the Work as-is and makes no representations or warranties of any kind concerning the Work, express, implied, statutory or otherwise, including without limitation warranties of title, merchantability, fitness for a particular purpose, non infringement, or the absence of latent or other defects, accuracy, or the present or absence of errors, whether or not discoverable, all to the greatest extent permissible under applicable law.{% endblocktrans %}</li>
+<li class="padding-{% start %}-normal padding-bottom-small">{% blocktrans trimmed %}Affirmer disclaims responsibility for clearing rights of other persons that may apply to the Work or any use thereof, including without limitation any person's Copyright and Related Rights in the Work. Further, Affirmer disclaims responsibility for obtaining any necessary consents, permissions or other rights required for any use of the Work.{% endblocktrans %}</li>
+<li class="padding-{% start %}-normal padding-bottom-small">{% blocktrans trimmed %}Affirmer understands and acknowledges that Creative Commons is not a party to this document and has no duty or obligation with respect to this CC0 or use of the Work.{% endblocktrans %}</li>
+</ol>
+  </li>
+
+      </ol>{# end section #}
```

Comment:
Ah, I think we are missing two closing <div> tags here

Reply:
Can we also add indentation to this file?

Reply:
Good :eye:.  I'll fix the closing tags and indent the file properly.